code/datums/trading/unique.dm

```diff
@@ -43,7 +43,7 @@
 		/obj/item/weapon/gun/energy/rifle/pulse         = TRADER_THIS_TYPE
 	)
 
-	blacklisted_trade_items = null
+	blacklisted_trade_items = list(/mob/living/carbon/human/monkey)
```

Comment:
Bad fix. Change line L35 to be: `/mob/living/carbon/human = TRADER_THIS_TYPE` instead.